streaming-binlog/src/main/java/org/openmrs/analytics/PipelineSink.java

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.openmrs.analytics;
+
+import java.util.Map;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+// A processor that sinks FHIR cloud and local
+// TODO implement sink to local
+public class PipelineSink implements Processor {
+	
+	private static final Logger log = LoggerFactory.getLogger(PipelineSink.class);
+	
+	// FhirStore
+	private static final FhirStoreUtil FHIR_STORE_UTIL = new FhirStoreUtil(System.getProperty("cloud.gcpFhirStore"));
```

Comment:
nit: This does not need to be `static`, in general avoiding `static` makes it easier to do unit-testing which has a side effect of making the code more modular. This can be resolved later, e.g., when we add unit-tests for this and/or proper dependency-injection.